api/src/lib.rs

```diff
@@ -493,6 +515,22 @@ pub extern "system" fn Java_com_horizen_schnorrnative_SchnorrSignature_nativeDes
     *sig_object
 }
 
+#[no_mangle]
+pub extern "system" fn Java_com_horizen_schnorrnative_SchnorrSignature_nativeIsValidSignature(
```

Comment:
is not used in Java `SchnorrSignature` class.

Reply:
This function is commented